ClassevivaPCTO/Views/SettingsPage.xaml.cs

```diff
@@ -302,7 +302,7 @@ private async void LanguageComboBox_OnSelectionChanged(object sender, SelectionC
                 var langProgressPerc = languageProgressObj.Data[0].TranslationProgress;
 
 
-                if (langProgressPerc != 100)
+                if (langProgressPerc != 100 && langProgressPerc != 0)
```

Comment:
perchè aggiungere langProgressPerc != 0 se già controlliamo che non sia 100% la traduzione?

Reply:
perché l'altro dialog non viene mostrato senza questa condizione

Reply:
non mi piace tanto l'altro dialog per bloccare l'utente al cambio lingua
la libreria per le traduzioni in tempo reale è quasi finita, togliamolo a questo punto